rwgame/GameWindow.cpp

```diff
@@ -0,0 +1,101 @@
+#include <png.h>
+#include <fstream>
+
+#include <core/Logger.hpp>
+#include "GameWindow.hpp"
+
+GameWindow::GameWindow() :
+	window(nullptr), glcontext(nullptr)
+{
+
+}
+
+
+void GameWindow::create(size_t w, size_t h, bool fullscreen)
+{
+	uint32_t style = SDL_WINDOW_OPENGL/* | SDL_WINDOW_ALLOW_HIGHDPI*/;
+	if (fullscreen)
+		style |= SDL_WINDOW_FULLSCREEN;
```

Comment:
While you are at it: `/* | SDL_WINDOW_ALLOW_HIGHDPI*/` dead code? Why was this there in the first place?

If you want to keep it: document why this is commented out and when it should be reintroduced. (Even in that case, just throw it away and create a issue about it if it's actually important)
If you don't want to keep it: throw it away.